### PR TITLE
Update l10n.js

### PR DESCRIPTION
**Pull request by chaals**
Use the whole word "Figure". 3 extra characters per picture isn't that expensive really.

### CODE REVIEW

File: src/core/l10n.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const base = {
   editors_note: "Editor's note",
   editors: "Editors:",
   example: "Example",
-  fig: "Fig. ",
+  fig: "Figure ",
```
Comment (marcoscaceres) [inline src/core/l10n.js]: I don't disapprove, but what motivates this change?

Comment (chaals) [inline src/core/l10n.js]: I started using `figure`s in a specification, and the abbreviation seemed unhelpful.

Admittedly this is close to the bike shed...

Comment (marcoscaceres) [inline src/core/l10n.js]: As my screenreader also doesn't know how to expand the "fig." abbr, I'm inclined to support this change.

**Comment by marcoscaceres** [comment]
Will release soon.

File: src/core/l10n.js (hunk 1 above)
Comment (chaals) [inline src/core/l10n.js]: (You can probably teach your screenreader to recognise it, but I think that's pushing the work to the wrong place :) ).

Thanks, BTW

**Pull request merged**